src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .trim()
+    .split(';')
```

Comment:
The trim method on the entire string is redundant here since you're trimming individual parts later in the code. You could remove it to make the code cleaner.